test/unittests/main_UnitTest.cpp

```diff
@@ -110,9 +129,26 @@ int main(int argC, char** argV)
   }
   else
     paParse(paArgs, 1, argV, 1, false);
+#else
+  // FIXME PR: With this change the following works:
+  //
+  // BUILD_UNITTEST/test/unittests/unitTest -t 0-255 -dbhost qa-bigdata-sth-01
+  //
+  // However, if we introduce gtest arguments, it breaks, e.g:
+  //
+  // BUILD_UNITTEST/test/unittests/unitTest -t 0-255 -dbhost localhost --gtest_filter=*.*
+  // ...
+  // option '--' is a boolean option - cannot have a value (gtest_filter=*.*)
+  //
+  // I don't know how to fix this... paArgs wisdom is required :)
+
+  paParse(paArgs, argC, (char**) argV, 1, false);
```

Comment:
With this change, paLib simply accepts the arguments and does nothing with them.
Later they are passed to InitGoogleMock and used there by gtest.
If there is ever any need for more args, just add them to the paArgs array.

Before, as paLib complained about unknown params and doing an exit(), we never got that far.